Lib/Src/CThermistor.cpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file CThermistor.cpp
+ */
+
+/*
+ *  Created on: Feb 15, 2022
+ *      Author: niuslar
+ */
+
+#include "CThermistor.h"
+#include "math.h"
+
+#define OUT_OF_RANGE_VALUE (999.9)
+
+CUartCom uart_temp(&huart2);
+CLog log_temp(&uart_temp, "Temp_sensor");
```

Comment:
Would it be possible to create the logger outside this class and then pass the pointer during construction? Make two constructors with and without logger. This way class can function even without logger.